yarn.lock

```diff
@@ -14699,6 +14769,30 @@ selfsigned@^1.10.4, selfsigned@^1.9.1:
   dependencies:
     node-forge "0.7.5"
 
+semantic-ui-css@^2.4.1:
```

Comment:
What was the reason for not going with the `semantic-ui-sass` port again?

Reply:
It just didn't feel like a good option. They don't support theming, and it's more or less just a sloppy port of the compiled LESS converted to CSS and then renamed to .scss files. With the limited usage that we're putting into it, it felt weird to pull in a fork of a project that was essentially just shipping semantic-ui-css. https://github.com/aniftyco/semantic-ui-sass/issues/1